python/pyspark/sql/dataframe.py

```diff
@@ -584,14 +584,14 @@ def printSchema(self, level: Optional[int] = None) -> None:
         .. versionchanged:: 3.4.0
             Supports Spark Connect.
 
-        .. versionchanged:: 3.5.0
-            Added Level parameter.
-
         Parameters
         ----------
         level : int, optional, default None
             How many levels to print for nested schemas.
 
+        .. versionchanged:: 3.5.0
```

Comment:
I think the indent is not right, you may refer to https://github.com/apache/spark/blob/7f724c3bc7567b0cddc09d5bed11b79879533368/python/pyspark/sql/dataframe.py#L1917-L1921

Reply:
Good spot! fixed. Thanks